### DISCUSSION TITLE
npm install bootstrap@5.3.2 Not working

### DISCUSSION THREAD

**Discussion by monirulislamkhan** [twbs/bootstrap#39662] [Q&A] [answered]
npm install bootstrap@5.3.2 Not working....
-------------------------------------------------------

### When i npm install bootstrap@5.3.2 hit Enter. The below message show me and not generate node_modules folder

up to date, audited 96 packages in 2s

13 packages are looking for funding
  run `npm fund` for details

5 vulnerabilities (1 moderate, 4 high)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.

**Kindly tell me the problem**

### What operating system(s) are you seeing the problem on?

Windows

### What version of Bootstrap are you using?

bootstrap@5.3.2

**Comment by julien-deramond** [comment, marked as answer]
Hello @monirulislamkhan 
This is not related to Bootstrap itself.
The message that you share is the normal one after running `npm i bootstrap@5.3.2`.
The problem seems to be linked to the creation of the `node_modules` directory. It might be a storage issue, a rights issue, a network issue linked to a VPN, how `.npmrc` is configured, etc. Difficult to say.
